src/components/common/CoreAppWrapper/CoreAppWrapper.tsx

```diff
@@ -33,7 +35,11 @@ export const AppWrapper: React.FC = ({ children }) => {
               <ThemeProvider>
                 <EtherPriceProvider>
                   <ArcxProvider>
-                    <_Wrapper>{children}</_Wrapper>
+                    <AnnouncementsProvider>
+                      <AnnouncementLauncher>
```

Comment:
there can be multiple `AnnouncementLaunchers` in the tree. currently there are 2:

- one here, to handle launching announcements from top level pages like the home page or /projects
- one wrapping v1 and v2 dashboards, to handle launching announcements dependent on ProjectContexts